store/copr/mpp.go

```diff
@@ -254,14 +254,14 @@ func (m *mppIterator) handleDispatchReq(ctx context.Context, bo *Backoffer, req
 	if originalTask != nil {
 		sender := NewRegionBatchRequestSender(m.store.GetRegionCache(), m.store.GetTiKVClient(), m.enableCollectExecutionInfo)
 		rpcResp, retry, _, err = sender.SendReqToAddr(bo, originalTask.ctx, originalTask.regionInfos, wrappedReq, tikv.ReadTimeoutMedium)
-		if err != nil && disaggregatedTiFlash {
```

Comment:
why remove this?

Reply:
In [(ss *RegionBatchRequestSender) SendReqToAddr](https://github.com/guo-shaoge/tidb/blob/bd76d19d9cfd1310af64217fa17bf2f6a53b3a7a/store/copr/batch_request_sender.go#L80), will check err and set RPCError. Also the returned err will be changed to nil if `onSendFailForBatchRegions` succeed.